simplex.py

```diff
@@ -170,10 +169,8 @@ def run(self, r_data=None):
                 data = calculate.main(self.args_ff)
                 ff.score = compare.compare_data(r_data, data)
                 opt.pretty_ff_results(ff)
+        self.new_ffs = sorted(self.new_ffs + [ff_copy], key=lambda x: x.score)
         wrapper = textwrap.TextWrapper(width=79)
```

Comment:
The variable wrapper should be moved near the code that uses wrapper, so move it down with the rest.